### PR TITLE
Add new design token for category colors, fix docs

### PR DESCRIPTION
**Pull request by ling1726**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

* Added new design token for category colors
* Fixed docs for category colors to display 'missing colors' which were always present

#### Focus areas to test

(optional)

### CODE REVIEW

**Review by assuncaocharles — APPROVED** [review]
LGTM

**Comment by assuncaocharles** [comment]
@ling1726 Please, add changelog

File: packages/fluentui/CHANGELOG.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Added `hoverBackground` and `paddingRight` variables to `treeTitleVariables` @assuncaocharles ([#15200](https://github.com/microsoft/fluentui/pull/15200))
 - Added `paddingRight` variable to `treeTitleVariables` @assuncaocharles ([#15206](https://github.com/microsoft/fluentui/pull/15206))
 - Added `LiveIcon` and `LiveOffIcon`. @TanelVari ([#15239](https://github.com/microsoft/fluentui/pull/15239))
+- Added new `borderActive` color design token to `categoryColorScheme` @ling1726 ([15717])(https://github.com/microsoft/fluentui/pull/15717)
```
Comment (assuncaocharles) [inline packages/fluentui/CHANGELOG.md:76]:
```suggestion
- Added new `borderActive` color design token to `categoryColorScheme` @ling1726 ([15717](https://github.com/microsoft/fluentui/pull/15717))
```

File: packages/fluentui/CHANGELOG.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Added `hoverBackground` and `paddingRight` variables to `treeTitleVariables` @assuncaocharles ([#15200](https://github.com/microsoft/fluentui/pull/15200))
 - Added `paddingRight` variable to `treeTitleVariables` @assuncaocharles ([#15206](https://github.com/microsoft/fluentui/pull/15206))
 - Added `LiveIcon` and `LiveOffIcon`. @TanelVari ([#15239](https://github.com/microsoft/fluentui/pull/15239))
+- Added new `borderActive` color design token to `categoryColorScheme` @ling1726 ([15717](https://github.com/microsoft/fluentui/pull/15717))
```
Comment (miroslavstastny) [inline packages/fluentui/CHANGELOG.md:76]:
```suggestion
- Added new `borderActive` color design token to `categoryColorScheme` @ling1726 ([#15717](https://github.com/microsoft/fluentui/pull/15717))
```

File: packages/fluentui/CHANGELOG.md [hunk 3]
```diff
@@ -100,6 +101,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Detect focus on body in use mode - UI builder @vyhnalekl ([#14954](https://github.com/microsoft/fluentui/pull/14954))
 - Fixed `ComponentTree` accessibility in UI builder @vyhnalekl ([#14967](https://github.com/microsoft/fluentui/pull/14967))
 - Adding screen reader labeling examples @kolaps33 ([#15146](https://github.com/microsoft/fluentui/pull/15146))
+- Fixed category color scheme dropdown to show all available colors @ling1726 ([#15717])(https://github.com/microsoft/fluentui/pull/15717)
```
Comment (miroslavstastny) [inline packages/fluentui/CHANGELOG.md:104]:
```suggestion
- Fixed category color scheme dropdown to show all available colors @ling1726 ([#15717](https://github.com/microsoft/fluentui/pull/15717))
```

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**